src/js/dp/search.js

```diff
@@ -54,15 +54,19 @@ if (searchContainer) {
     } else {
       const fetchedDom = new DOMParser().parseFromString(responseText, "text/html");
 
-      const resultsCount = fetchedDom.querySelector(".search__summary__count").innerText;
-      const noResults = resultsCount.startsWith("0");
-      const noResultsMessage = document.querySelector('#results-zero')
-
-      if (noResults) {
-        if (noResultsMessage) noResultsMessage.classList.remove('hide');
-        searchContainer.querySelector("#results > ul").innerHTML = '';
-        searchContainer.querySelector(".search__pagination").innerHTML = '';
-        searchContainer.querySelector(".search__summary__count").innerText = '0';
+      let resultsCount = 0;
+      const searchPrompt = fetchedDom.querySelector(".search__form--no-results");
+      if (!searchPrompt) {
+        resultsCount = fetchedDom.querySelector(".search__summary__count").innerText;
+      }
+
+      const noResultsMessage = document.querySelector("#results-zero");
+
+      if (resultsCount === 0) {
+        if (noResultsMessage) noResultsMessage.classList.remove("hide");
```

Comment:
Not having the open and closing braces makes it harder to understand the intention here...
Should the assignment on lines `67-69` be part of the guard clause on line `66`?

Reply:
Have resolved the brackets. 

I think the original intent of the guard clause at 66 was in case the noResultsMessage didn't exist and so couldn't be manipulated. Whereas the other elements had been in place for a long time in the code base.